src/content/github/common/utils.ts

```diff
@@ -0,0 +1,12 @@
+// Note that we're guaranteed to be on a github page due to manifest.json,
+// so these checks don't need to include that.
+const prUrlRegex = /\/[^\/]+\/[^\/]+\/pull\/\d+\/files.*/
+const fileUrlRegex = /\/[^\/]+\/[^\/]+\/blob\/[^\/]+\/.*/
+
+export function isFileUrl(url: string): boolean {
```

Comment:
nit: Maybe we could give a single example of what the regex will typically be matching in each case? I'm not a regex aficionado though so maybe others wouldn't have a tough time here.

Like ex: 
/user/repo/blob/main/file.txt is a fileURL
/user/github/main/file.txt is not a fileURL

Reply:
will do this good idea